### DISCUSSION TITLE
🤔 [饼图提供字段显示标题吗]

### DISCUSSION THREAD

**Discussion by zyk2891** [antvis/G2Plot#3572] [Q&A] [answered]
### 🐛 Question description [Please make everyone to understand it]
柱状图axis有title属性，但饼图没有axis，那只能自建div显示标题吗？在图例(legend)左右分布的时候，就不能简单通过text-align: center控制居中

![image](https://user-images.githubusercontent.com/43950678/216223218-1fe9607b-866e-46ab-bd6c-7d82f91306be.png)

**Comment by hustcc** [comment, marked as answer]
是的，当前版本，对于标题，只能外部自建。关于图表标题，我们在 [G2 5.0 新版本](https://g2.antv.antgroup.com/spec/component/title)中考虑进去了。
